server/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express")
+const path = require('path')
+const fs = require('fs')
+const match = require('path-to-regexp').match
+const fetch = require('node-fetch')
+const composeHeadData = require('./util').composeHeadData
+const composeJSONLD = require('./util').composeJSONLD
+
+const API_BASE_URL = process.env.REACT_APP_API_BASE_URL || 'http://localhost:2503'
+const PORT = 3000 
+
+const app = express()
+
+app.use(express.static(path.join(__dirname, "..", "build")))
+app.use(express.static(path.join(__dirname, "..", "public")))
+
+var server = app.listen(PORT, () => {
+  console.log(`server started on port ${PORT}`)
+})
+
+// reservedUsernames are a list of words that may be mistaken for usernames
+// during a url regex match of `/:username/:name`, because of how our routes
+// are formed
+// must be updated if routes change, particularly routes with 2 segments as
+// those are the routes at risk for being incorrectly matched
+const reservedUsernames = ['ipfs', 'login', 'workflow'] 
+
+
+// reservedNames are a list of words that may be mistaken for dataset names
+// during a url regex match of `/:username/:name`, because of how our routes
+// are formed
+// must be updated if routes change, particularly routes with 2 segments as
+// those are the routes at risk for being incorrectly matched
+const reservedNames = ['following']
+
+const indexPath = path.join(__dirname, "..", "build", "index.html")
+
+fs.access(indexPath, fs.constants.F_OK, (err) => {
+  if (err !== null) {
+    console.log(`File ${indexPath} does not exists. App must be built before attempting to host it.`)
```

Comment:
bump this up to `console.error`, add the err itself to the log message, change `exists` -> `exist`